src/hashes.txt

```diff
@@ -400,3 +400,144 @@ e6747c1a4c460fd40a05a0f419db1dc139c33e72447ac36f96d59451b213a4c7 scriptaculous.1
+776f0615c1a084a93f849b71d4147e5350f1a51e70ee01288a14e68a0be43da0 angularjs.1.2.16.angular.js
+854e6d106ea8724b7ae3a134aa983d33defdac7bb19c143ee1e4b2b77474b90a angularjs.1.3.6.angular.js
+acce50bf76c99d3c3e7dd8d5f4a53b5b9ebe1d9224be924e7cce2ed462d286ba angularjs.1.3.3.angular.js
+4a09bb4fa93c1a5032bf71df14ee196a84eb1631e93301f20dbc5249a974f74d angularjs.1.3.9.angular.js
+6f7e8a3d47b9919bd8d2139fba72af7adf88f035c302574662b27b1e96d2a1f9 angularjs.1.3.14.angular.js
+7ef18bebed6b6bd70d52b2ab692e95696dc5fc5fa451f24b2d36b49e46db403c angularjs.1.3.17.angular.js
+8425489c06275464b9b7b116736a8b6b59bf03cda527bc46b156860615cddf6d angularjs.1.3.19.angular.js
+a1e21be338c49038fa4d7d341dd751cd0fe69d89356a5ba8aa917742441a41fa angularjs.1.3.2.angular.js
+ba1d8eccaa8b2ec6ca0b6c6e3f9579207044e133fbc463b608492ef0fde5968b angularjs.1.2.24.angular.js
+11712f419e6efa6f45c323f5243f288786c3c6002f1b5715d8b06eb4165a9c77 angularjs.1.3.5.angular.js
+4d892525d5139e768096b26de39ae9f404ff49c99a8db1dca8ffd145a4bb24fd angularjs.1.3.16.angular.js
+aba7844e248289dfd9f7a3810998f3a69a64354f366f87c49f2c4414dbda3238 angularjs.1.3.12.angular.js
+64d11ab9e3de6672d412ae00466287b815a0c97a92460501c3c40f085146c570 angularjs.1.3.8.angular.js
+d3b659ff6489c82bc689238d2a4c7f8155a92cef8028b183f4d016bb3947c396 angularjs.1.3.13.angular.js
+95fafe1812c623e8d1ea67aac761ae8ffbe09cca0e45b602b15a43d4f73ba924 angularjs.1.3.15.angular.js
+c82d36962acdb9eae9212ffaa2fee14cb011846bd5db2acf990094583baea04d angularjs.1.3.4.angular.js
+5e2e1babaa1003141c44760239f7df505fbe61484bb5dd350f583c5b85a1bc66 jquery.1.6.3.jquery.js
+fa49c78b49f1c1bf6386e2dc6b9bc7f159453d1c765dc693597defa45416d9bc jquery.1.7.2.jquery.js
+ad8a08bc7ac215669f4a6b6380aa13e4069f55de12ee7772037c605d47d53cb2 angularjs.1.2.2.angular.js
+1444e75e04f26b9cdaff9f9eb25379b947a25469d283c458b583bda4e9e40e87 angularjs.1.5.5.angular.min.js
+47b04b437a89b47e25a1595f0a58ef1555a4ef3fe595c00ba5a1451fad625fa1 angularjs.1.5.5.angular.js
+7a0967e52f8765b7280e6c6bd05d47f3d48e2df574fc81278179fbb5ab7810fb angularjs.1.1.0.angular.min.js
+ed48da7c0f6ce4a0cb4d042d533fec7b549dd50e0a2d2d69bdbdefb8e6c70ef0 angularjs.1.1.2.angular.min.js
+197f9aa6d87bcc1845e7654cc9042dcfee7b4c6c5d1cff348edbfcb528cd2bb5 angularjs.1.1.0.angular.js
+fe0fe482bfb9dcd4a8f3d719636ae96d97187e915ebfb4a2f734add01d5e1620 angularjs.1.1.2.angular.js
+f7948aa0ccb702ee7707e2fd4182cbfa21ba89ff38b53fb6ffdfb62982a7b710 angularjs.1.2.26.angular.js
+8e5fa036a92cb5a9fa90c648b3fed2677fceb3fb9b281ae5f5f2549b0050c895 angularjs.1.5.4.angular.min.js
+d084e9f50f8c16af6a501fa74ac622f6a260a8f8997870e2479aa595411914f5 angularjs.1.5.4.angular.js
+668b046d12db350ccba6728890476b3efee53b2f42dbb84743e5e9f1ae0cc404 jquery.1.12.4.jquery.min.js
+430f36f9b5f21aae8cc9dca6a81c4d3d84da5175eaedcf2fdc2c226302cb3575 jquery.1.12.4.jquery.js
+05b85d96f41fff14d8f608dad03ab71e2c1017c2da0914d7c59291bad7a54f8e jquery.2.2.4.jquery.min.js
+893e90f6230962e42231635df650f20544ad22affc3ee396df768eaa6bc5a6a2 jquery.2.2.4.jquery.js
```

Comment:
When you run the update command it fetches lots of new versions for all libs; filtering them out seemed silly.